lib/menu.rb

```diff
@@ -0,0 +1,29 @@
+class Menu
+
+  attr_reader :menu_list
+
+  def initialize
+    @menu_list = { 'Fish & Chips' => 4, 'Burger' => 3, 'Shawarma' => 5 }
```

Comment:
Maybe start with an empty list, so the class can be used for any restaurant, not just a Fish and Chip restaurant?